examples/hello_ocean.jl

```diff
@@ -0,0 +1,35 @@
+# # Hello, ocean!
+#
+# > How inappropriate to call this planet Earth, when it is quite clearly _Ocean_.
+#
+#   --Arthur C. Clark
+
+using Oceananigans, Oceananigans.Grids, Plots
+
+grid = RegularCartesianGrid(size = (1, 64, 64),
```

Comment:
why not `x` and `z`? I think that's more natural

Reply:
because using y, z generalizes to GPU

Reply:
and `x, z` does not?

Reply:
no, not if x is bounded

Reply:
you are saying that
```Julia
grid = RegularCartesianGrid(size = (1, 64, 64),
                            x = (-4, 4), y = (0, 1), z = (-4, 4),
                            topology = (Bounded, Periodic, Bounded))
```
won't run on GPU?

Reply:
Yeah `topology = (Bounded, Periodic, Bounded)` won't even run on the CPU right now for the same reason as https://github.com/CliMA/Oceananigans.jl/issues/1007.

I'm working on supporting all topologies in the `ali/pressure-solver-tlc` branch so `topology = (Bounded, Periodic, Bounded)` works on the CPU there, but still needs more work and TLC for the GPU to support all topologies.